TASVideos/wwwroot/js/show-more.js

```diff
@@ -37,4 +37,10 @@ function registerShowMore(content) {
 
     show.querySelector('a').onclick = () => false;
     hide.querySelector('a').onclick = () => false;
+
+    content.onclick = function (e) {
+        if (e.target.tagName === "A" && e.target.getAttribute('href')?.startsWith('#') && content.style.overflowY === "hidden") {
```

Comment:
Can we consistently use single quotes?

Reply:
Sure! But there are other instances of double quotes in this file that I haven't touched.